dhall/benchmark/parser/Main.hs

```diff
@@ -96,4 +101,7 @@ main = do
         , benchExprFromText "Block comment" ("x {- " <> T.replicate 1000000 " " <> "-}")
         , benchExprFromText "Deeply nested parentheses" "((((((((((((((((x))))))))))))))))"
         , benchParser prelude
+        , env cpkgExample $ \e ->
+            benchNfExprFromText "CPkg/Text" e
```

Comment:
One inconsistency with the benchmarks above is that this one evaluates to NF instead of just WHNF.

(The others actually only evaluate the `(Either Error Expr)` to WHNF!

I'm unsure about the pros and cons here. One con might be that the derived `NFData` instance might be rather costly itself…

What do you think?

Reply:
I prefer `nf`, personally - though one needs perspective and benchmarking all stages of the pipeline helps! 

In my experience, it is easy to go awry with `whnf`; I haven't had much problem being misled by `nf` since one can simply benchmark bigger chunks of the program!